nginx.conf

```diff
@@ -0,0 +1,59 @@
+# User and worker settings
+user www-data;
+worker_processes auto;
+pid /run/nginx.pid;
+include /etc/nginx/modules-enabled/*.conf;
+
+# Events block for handling connections
+events {
+    worker_connections 768;
+    # multi_accept on;
+}
+
+# HTTP block for handling HTTP requests
+http {
+    # Basic Settings
+    sendfile on;
+    tcp_nopush on;
+    tcp_nodelay on;
+    keepalive_timeout 65;
+    types_hash_max_size 2048;
+    server_tokens off;
+
+    # Include MIME types
+    include /etc/nginx/mime.types;
+    default_type application/octet-stream;
+
+    # SSL Settings
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2; # Dropping SSLv3, ref: POODLE
+    ssl_prefer_server_ciphers on;
```

Comment:
mo ssl during the tests

Reply:
So, I have to remove these two lines and don't have to add anything, right?

Reply:
yes, anything with ssl won't be used